timelock-server/versions.lock

```diff
@@ -1336,7 +1334,7 @@
             ]
         },
         "com.fasterxml.jackson.module:jackson-module-jaxb-annotations": {
-            "locked": "2.7.8",
+            "locked": "2.6.7",
```

Comment:
This is a minor version rollback (2.7.8 -> 2.6.7). Is this concerning? Surely we used 2.7.x for a reason previously?